app/src/main/res/xml/quran_preferences.xml

```diff
@@ -147,29 +147,16 @@
   </PreferenceCategory>
 
   <PreferenceCategory
```

Comment:
Is it possible to handle `PreferenceCategory` clicks and launch advanced settings thereof, and get rid of additional `Preference` below?